client/src/components/Lobby.svelte

```diff
@@ -44,8 +63,22 @@
       <h3>Players: {lobby.players.length}</h3>
     </div>
     <SelectAvatar {lobby} {lobbyCode} />
-    <div class="start">
-      <Button on:click={() => startGame({ code: lobbyCode })}><Label>Start Game</Label></Button>
+    {#if auth.currentUser?.uid === lobby.uids[0]}<div class="actions">
+        <Button on:click={() => start()}><Label>Start Game</Label></Button>
+      </div>
+    {/if}
```

Comment:
I don't like how the `<div>` is on the same line as `{#if}` but the `</div>` is on a diffrent line from `{/if}`